python/lsst/pipe/supertask/activator.py

```diff
@@ -131,6 +132,34 @@ def activate(self):
         """
         pass
 
+def get_inheritance(stask_class, subclasses=[]):
```

Comment:
I was calling always with empty [] to avoid that :). I've changed to the above. Thanks